.github/workflows/release-packages.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
           asset_content_type: application/x-deb
       - name: Slack notification of CI status
         uses: rtCamp/action-slack-notify@v2
+        if: success() || failure()
```

Comment:
:shouldn't the message change if the job has failed?

Reply:
It will, yes.

This is different - it's to ensure that the step will run on either success or failure of the previous steps.

Because without this, right now what happens is that if any of the previous steps fails (say, the tests fail) all of the next steps in the job get skipped, so we don't get to see the notification.

I got this from https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions if you want to have a look.

There's also `always()` but this runs even if a job is cancelled, which is probably not what we want.